apps/client/app/components/PurchaseOrderForm.tsx

```diff
@@ -0,0 +1,167 @@
+// This is the only client component so far in the application due to the need for interactivity
+'use client'
```

Comment:
This is the component that is used for both creating and editing purchase order records. It made sense to use a client side rendering here due to the amount of user interactivity. 

The approach I took in general was to use SSR unless there was a need for CSR.